charts/seed-bootstrap/charts/loki/templates/kube-rbac-proxy-ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.rbacSidecarEnabled }}
+apiVersion: {{ include "ingressversion" . }}
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
+    nginx.ingress.kubernetes.io/configuration-snippet: "proxy_set_header X-Scope-OrgID operator;"
+  name: loki
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ toYaml .Values.labels | indent 4 }}
+spec:
+  tls:
+  {{- range .Values.ingress.hosts }}
+  - secretName: {{ required ".secretName is required" .secretName }}
+    hosts:
+    - {{ required ".hostName is required" .hostName }}
+  {{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+  - host: {{ required ".hostName is required" .hostName }}
+    http:
+      paths:
+      - backend:
+          {{- if semverCompare ">= 1.19-0" $.Capabilities.KubeVersion.GitVersion }}
+          service:
+            name: loki
+            port:
+              number: 8080
```

Comment:
If it is actually the `kubeProxy.port` you can just use {{.Values.kubeProxy.port}}
Also there are some more places with hardcoded port. Check them

Reply:
In a loop such a this one ` {{- range .Values.ingress.hosts }}` a `.Values...` must exists in `.Values.ingress.hosts` so for your example a have to have `Values.ingress.hosts[n].Values.kubeProxy.port`